single_kernel_mongo/managers/ldap.py

```diff
@@ -171,6 +171,10 @@ def save_certificates(self, chain: list[str] | None) -> None:
 
         self.workload.write(self.workload.paths.ldap_certificates_file, full_chain)
 
+        with open(TRUST_STORE_PATH / TrustStoreFiles.LDAP.value, mode="w") as fd:
+            # boto3 client will need the certificate on the node that runs the command
+            fd.write("\n".join(full_chain))
```

Comment:
We should go through the `workload` component for k8s compatibility, no?

Reply:
Specifically no as this code runs on the charm: we write this file in the pod that runs the charm code, not the pod that runs the service.
The workload reference remote containers in the single kernel implementation.
This code is similar to [the one for certs for backup](https://github.com/canonical/mongo-single-kernel-library/blob/6/edge/single_kernel_mongo/managers/backups.py#L258-L260)